### ISSUE TITLE
Current Weather NOT WORKING!! Please HELP!! [Bug: Current]

### ISSUE THREAD

**Issue by jayinc** [SaberAkiyama/SAWeatherCast#33]
Used to work fine till a few months ago. 
But now all 4 in files under the Current section DO NOT WORK!!
I input the code for Vancouver, Bc which is "CAXX0518" although it gives me the weather forecast on refresh, the current weather widget remains unresponsive and blank. 

<img width="302" alt="image" src="https://user-images.githubusercontent.com/12969720/213357649-fa6757ec-2068-4bcd-bef1-9d77ee0fc51b.png">

I also tired to input the full weather code from weather.com URL for Vancouver, BC which is "31fa7b6da0d67b704f144e94301d921023ab7bfe5a7de83b036bb1a652989d23", since someone had mentioned it in an closed issue for forecast, but that didn't solve the issue either. 
Hoping someone can offer a quick fix solution to patch the bug.

**Expected behavior**
A clear and concise description of current weather along with other info as seen below (in °F).
<img width="407" alt="image" src="https://user-images.githubusercontent.com/12969720/213357983-e5251d47-02de-4d0c-ae08-fc9641c78965.png">

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro 22H2 (build 22621.1105) 64-bit - English (1033)
 - Rainmeter version: Rainmeter 4.5.17.3700 (64-bit)
 - Language: English (1033)
 - Build time: 2022-12-24 20:33:24

**Additional context**
The problem exists for both °C & °F

**Comment by SaberAkiyama** [comment]
Right-click on the current skin, hover over "Custom skin actions" & left-click "Open Settings panel."

Inside Settings, left-click "Language," left-click "English (World)" & left-click "Apply Changes" to refresh the current skin.
